ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,276 @@
+
+Instructions for setting up Gmail filters can be found [here](#setting-up-gmail-filters).
+
+## 2. Ensure the issue contains basic information
+
+Before triaging an issue very far, make sure that the issue's author provided the standard issue information. This will help you make an educated recommendation on how to categorize the issue. The Grafana project utilizes [GitHub issue templates](https://help.github.com/en/articles/creating-issue-templates-for-your-repository) to guide  contributors to provide standard information that must be included for each type of template or type of issue.
+
+### Standard issue information that must be included
+
+Given a certain [issue template]([template](https://github.com/grafana/grafana/issues/new/choose)) have been used by the issue author or depending how the issue is perceived by the issue triage responsible, the following should help you understand what standard issue information that must be included.
+
+#### Bug report?
+
+Should explain what happened, what was expected and how to reproduce it together with any additional information that may help giving a complete picture of what happened such as screenshots, [query inspector](https://community.grafana.com/t/using-grafanas-query-inspector-to-troubleshoot-issues/2630) output and any environment related information that's applicable and/or maybe related to the reported problem:
+- Grafana version
+- Data source type & version
+- Platform & OS Grafana is installed on
+- User OS & Browser + versions
+- Using docker + what environment
+- Which plugins
+- Configuration database in use (sqlite, mysql, postgres)
+- Reverse proxy in front of Grafana, what version and configuration
+- Non-default configuration settings
+- Development environment like Go and Node versions, if applicable
+
+#### Enhancement request?
+
+Should explain what enhancement or feature that the author wants to be added and why that is needed.
+
+#### Accessibility issue?
+
+This is a mix between a bug report and enhancement request but focused on accessibility issues to help make Grafana improve keyboard navigation, screen-reader support and being accessible to everyone. The report should include relevant WCAG criteria, if applicable.
+
+#### Support request?
+
+In general, if the issue description and title is perceived as a question no more information is needed.
+
+### Good practices
+
+To make it easier for everyone to understand and find issues they're searching for it's suggested as a general rule of thumbs to:
+
+* Make sure that issue titles are named to explain the subject of the issue, has a correct spelling and and doesn't include irrelevant information and/or sensitive information.
+* Make sure that issue descriptions doesn't include irrelevant information, information from template that haven't been filled out and/or sensitive information.
+* Make changes to title and description after best effort or request suggested changes by adding a comment.
```

Comment:
`after best effort` sounds kinda weird or I do not sure exactly what that means I assume it should be something like `Do your best effort to change title and description or ...`?